### PR TITLE
Nick dev

### PR DESCRIPTION
**Pull request by ghost**
Nick dev is 13 commits ahead of your main. Some of the changes are from ESSAdev as Nick dev was created from that branch.

### CODE REVIEW

**Comment by wagnerds** [comment]
Request received.   Will review and post updated branches as quickly as possible.  Changes will be incorporated into ESSAdev and FSMCdev branches.

File: volume/volinit.f [hunk 1]
```diff
@@ -530,6 +530,8 @@ SUBROUTINE VOLINIT(REGN,FORST,VOLEQ,MTOPP,MTOPS,STUMP,DBHOB,
       IF(STUMP.LE.0) STUMP = 1.0
 C  calc stump DIB
       HTUP = STUMP
+C  this line traps cases where the calls to calcdia2 do not return a DIB.
+      DIB = -1.0
```
Comment (wagnerds) [inline volume/volinit.f:534]: Issue is addressed in seperate upcoming commit, initializing DIB to 0, as well as initializing other variables and updates to this file

**Comment by ghost** [comment]
Yes, but unit to -1 is a better way to catch situations where dib is not
set by the routine being call.

**Comment by wagnerds** [comment]
This pull request has been reviewed and incorporated into the GitHub enterprise repository and have been pushed back out to the open-source side under a new branch called ‘development’, which should now be your main source for creating new branches or staying up to date with what we have been working on.

**Pull request closed without merging**